lib/BaseStripeClient.php

```diff
@@ -163,6 +163,42 @@ public function request($method, $path, $params, $opts)
         return $obj;
     }
 
+    /**
+     * Sends a raw request to Stripe's API. This is the lowest level method for interacting
+     * with the Stripe API. This method is useful for interacting with endpoints that are not
+     * covered yet in stripe-php.
+     *
+     * @param 'delete'|'get'|'post' $method the HTTP method
+     * @param string $path the path of the request
+     * @param array $params the parameters of the request
+     * @param array $opts the special modifiers of the request
+     *
+     * @return \Stripe\ApiResponse
+     */
+    public function rawRequest($method, $path, $params, $opts)
+    {
+        if ('post' !== $method && null !== $params) {
+            throw new Exception\InvalidArgumentException('Error: rawRequest only supports $params on post requests. Please pass null and add your parameters to $path');
```

Comment:
I like the direction of this error message, but `Please pass null` could be more descriptive. Maybe something like

> raw $method requests don't support params (expected $params to be `null`). Either make a POST request instead or pass `null` to the params argument

Feel free to tweak that further though!